testnet/stacks-node/src/neon_node.rs

```diff
@@ -263,7 +263,7 @@ fn spawn_peer(mut this: PeerNetwork, p2p_sock: &SocketAddr, rpc_sock: &SocketAdd
             let network_result = this.run(&burndb, &mut chainstate, &mut mem_pool, Some(&mut dns_client), download_backpressure, poll_ms)
                 .unwrap();
 
-            if network_result.has_data_to_store() {
+            if network_result.has_blocks() || network_result.has_microblocks() {
```

Comment:
Why are we excluding the case `network_result.has_transactions()`, covered by the initial call to `has_data_to_store()`?

Reply:
Got it, thank you!